src/legacy/core_plugins/kibana/public/dashboard/dashboard_app.html

```diff
@@ -10,6 +10,7 @@
       <div ng-show="chrome.getVisible()" class="fullWidth" data-transclude-slot="bottomRow">
```

Comment:
@maryia-lapata, I've updated the line and removed dashboard-title from that line as well as from another one that is below. Thanks for the hint!